### DISCUSSION TITLE
notes/14.0.0/client/index

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#64] [Q&A]
# notes/14.0.0/client/index

https://help.hcltechsw.com/notes/14.0.0/client/index.html

**Comment by priyanvishnu2011** [comment]
Need for MAC os

**Reply by digihernandez** [reply]
While there isn't a separate document for Notes on Mac, it is integrated into the main document. You can find references by searching for "Macintosh," where specific topic titles will end with "(Macintosh)." Additionally, individual paragraphs in other topics will begin with "Macintosh OS X users:" We only document aspects that differ from Windows. If a user is looking for something specific, they should specify what that is. Please note that the link provided only directs to the welcome page.
